templates/common-init-files/secret.yaml

```diff
@@ -22,6 +22,7 @@ stringData:
     net.bridge.bridge-nf-call-iptables  = 1
     net.bridge.bridge-nf-call-ip6tables = 1
     net.ipv4.ip_forward                 = 1
+    net.ipv6.conf.all.forwarding        = 1
```

Comment:
can this be set if dual-stack is not enabled?

Reply:
Yeah, I figured to add it regardless since we already set `net.bridge.bridge-nf-call-ip6tables = 1` regardless of dual-stack as well